tests/test_lib_events.py

```diff
@@ -571,7 +571,6 @@ def test_02_failscript(self):
         d = "{0!s}/tests/testdata/scripts/".format(d)
         t_handler = ScriptEventHandler(script_directory=d)
         self.assertRaises(Exception, t_handler.do, script_name, options=options)
-        remove_token("SPASS01")
```

Comment:
What changed here?
If the token does not exist, we can catch an exception.

OK, due to the failing script, the token was not created? - right it is raise-error=True.
Maybe it is a good idea to realy check in this case, that the originial request also failed.

   self.assertRaises(Exception, remove_token, "SPASS01")

Reply:
I think this is a new test from #1307. My understanding is that the token "SPASS01" does not actually exist from the perspective of privacyIDEA (because we only test the ScriptEventHandler), so no need to try to remove it :-) 
I think I've already done the same for ``test_01_runscript``.

Reply:
You are right. The token really does not exist!
So this is fine!